smoothie.py

```
@@ -1,156 +1,80 @@
+    PurchaseItem('Large', 2.00),
+    PurchaseItem('Galactic', 100.00),
+]
 
-def pose_question_with_costs(question, option1, cost1, option2, cost2, option3, cost3, option4, cost4):
+Toppings = [ 
+    PurchaseItem('No Topping', 0.00),
+    PurchaseItem('Caramel Sauce', 0.50),
+    PurchaseItem('Brownie Fudge', 3.00),
+    PurchaseItem('Baked Beans', 1.75)
+]
+
+def getResponse(question, items):
     print(question)
-    print('1.', cost1, '       ', option1)
-    print('2.', cost2, '       ', option2)
-    print('3.', cost3, '       ', option3)
-    print('4.', cost4, '       ', option4)
-    
-    
-    
-        
-    
-def order():
-    question = "Hello and welcome to Big Ep's smoothie shop! \nWhat kind of smoothie would you like today? \nHave you tried our new Onion Toffe smoothie?"
-    pose_question_with_costs(question, SMOOTHIE1_NAME, SMOOTHIE1_COST, SMOOTHIE2_NAME, SMOOTHIE2_COST, SMOOTHIE3_NAME, SMOOTHIE3_COST,
- SMOOTHIE4_NAME, SMOOTHIE4_COST)
-    
-    
-    
-#Now we need to let them choose their smoothie before proceeding 
+    entry = 1
+    for item in items:
+        print(entry, item.name, ' $%.2f' % item.cost)
+        entry += 1
     choice = int(input('Which option would you like? (1-4): '))
-    if choice == 1:
-        selection = SMOOTHIE4_NAME
-        print('Sorry! We are all out of', SMOOTHIE1_NAME, '\nYou will be served Onion Toffee instead.')
-    elif choice == 2:
-        print('Sorry! We are all out of', SMOOTHIE2_NAME, '\nYou will be served Onion Toffee instead.')
-        selection = SMOOTHIE4_NAME
-    elif choice == 3:
-        print('Sorry! We are all out of', SMOOTHIE3_NAME, '\nYou will be served Onion Toffee instead.')
-        selection = SMOOTHIE4_NAME
-    elif choice == 4:
-        print('You have selected ', SMOOTHIE4_NAME)
-        selection = SMOOTHIE4_NAME
-    else:
-        selection = 0
+    if (choice < 1) or (choice > 4):
         print('That is not a valid option.')
         quit()

```

It would be better to throw an exception here rather than calling quit().